### DISCUSSION TITLE
Ideas and Feedback

### DISCUSSION THREAD

**Discussion by The-Viper-One** [The-Viper-One/PsMapExec#10] [Ideas]
Discussion thread for any general improvements, idea and features you might want adding :)

**Comment by ThatTotallyRealMyth** [comment]
I think two a super awesome feature that would be incredible to have is a module for coercion attacks. Similar to nxc, it could just scan a single target, all targets or whatever you chose and show you which types of attacks it is vulnerable to. Then allowing you the option to have pme either auto try to abuse all of them and give it the address you're listening on or it provides you the option to specify which of the exploits you want it to fire at the vulnerable system.

**Reply by The-Viper-One** [reply]
Hey @ThatTotallyRealMyth 

Thanks for the suggestion.

A Coercion method is on the roadmap for PME. I cant say when,  but it should hopefully be this year.  

The `-Module` SessionExec can be used for direct user coercion on remote system (Providing you are admin).

- https://github.com/The-Viper-One/PsMapExec/wiki/04-%E2%80%90-Modules#sessionrelay
